lib/liquid/block.rb

```diff
@@ -1,9 +1,16 @@
 module Liquid
   class Block < Tag
-    IsTag             = /^#{TagStart}/o
```

Comment:
That explains the merge conflict message. Updating my fork now as your right its from 2013.